app/config/config/parsers.js

```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const chalk = require('chalk');
+
+const { mutateByPath, parseChalkTemplate } = require('../../utils');
+const { DEFAULT_CONFIG_PATH } = require('../../utils/constants');
+
+const parseFileConfig = (configPath) => {
+  const resolvedPath = path.resolve(configPath || DEFAULT_CONFIG_PATH);
+
+  try {
+    // eslint-disable-next-line global-require
+    const config = require(resolvedPath);
+
+    console.log(chalk`{green info:} loaded config file "${resolvedPath}"\n`);
+
+    return config;
+  } catch {
+    if (configPath) {
```

Comment:
Maybe also add an error message, when the server tries to load the default config and due to an error it was not created, or the user deletes it

Reply:
Don’t get what you mean. There is no default config. There is only a example config which the user needs to copy to its own or adjust the config path to the example (which is a bad idea). The default config path is set to `./vocascan.config` which is a non existent file if you just cloned the repo. Do you need to create that file at your own. The warning will only be logged, if the config file path was actively set by the administrator. And I also planned to add a cmd flag to set the config file later in the cli pr. I also planed a little Startup message which showes the version, vocascan in ascii art or something and some other information like which config file is used which will replace the console log a few lines above in the try statement.

Reply:
`> The default config path is set to ./vocascan.config which is a non existent file if you just cloned the repo.`

This is the part I meant. If you just clone the repo no default config file is existing. But as far as I see, the error message, is only triggered when the admin set the path directly, which is not the case on the described scene. This is the reason why the message is not logged, although no config file could be opened.